### PR TITLE
FJ-22/--converts from ftp to ftps

### PR DESCRIPTION
**Pull request by JeremyGibson**

### CODE REVIEW

File: nc/data/download_from_nc.py [hunk 1]
```diff
@@ -31,26 +38,20 @@ def nc_download_and_unzip_data(destination, prefix='state-'):
...
         nc_data_directory = '/TSTOPextract'
-
-        # Note: NC documents show FileZilla set up to use explicit FTP over TLS
-        #       if available (like FTP_TLS), but the server doesn't currently
-        #       support it.
-        ftp = FTP(nc_data_site)
-        ftp.login(nc_data_user, nc_data_password)
-        ftp.cwd(nc_data_directory)
+        ftps = ftps_connect(nc_data_site)
+        ftps.prot_p()
```
Comment (copelco) [inline nc/data/download_from_nc.py:45]: What's prot_p for?

File: nc/data/download_from_nc.py [hunk 2]
```diff
@@ -13,6 +14,12 @@
 def show_ftp_listing(s):
     logger.debug(s)
 
+def ftps_connect(host):
+    ftps = FTP_TLS() 
+    ftps.ssl_version = ssl.PROTOCOL_SSLv23 
+    ftps.connect(host) 
+    ftps.login(os.environ.get('NC_FTP_USER'), os.environ.get('NC_FTP_PASSWORD'))
```
Comment (copelco) [inline nc/data/download_from_nc.py:21]: I know it used to be this way, but I'd prefer if we moved all 3 ``NC_*`` variables to settings to match our general workflow for handling env variables.

File: nc/data/download_from_nc.py (hunk 1 above)
Comment (JeremyGibson) [reply nc/data/download_from_nc.py:45]: It actually secures the connection. Without this call the connection fails.  Maybe there is a way to integrate it into the tls constructor.

Comment (JeremyGibson) [reply nc/data/download_from_nc.py:45]: https://docs.python.org/3/library/ftplib.html#ftplib.FTP_TLS

`Securing the data connection requires the user to explicitly ask for it by calling the prot_p() method. `

**Pull request merged**